js/main-menu.js

```diff
@@ -63,7 +63,8 @@
    * @return {boolean}
    */
   function isQuickSearchOnGoing () {
-    return !!$('#sort_name_navigation').val().trim();
+    var searchValue = $('#sort_name_navigation').val() || '';
+    return !!searchValue.trim();
```

Comment:
@GinkgoFJG please add empty line before this to make it look like
```js
function isQuickSearchOnGoing () {
  var searchValue = $('#sort_name_navigation').val() || '';

  return !!searchValue.trim();
}
```

Reply:
All set.